### PR TITLE
apply page

### PR DESCRIPTION
**Pull request by kevinmram**
finished (v1.0) of the apply page - 'hero' component, according to the Figma.
However, it isn't exactly as the design intended- will make any necessary changes/updates.

image

Furthermore, I started an initial implementation of the 'timeline' component, inserted below the 'hero' component.

### CODE REVIEW

**Comment by oscarwang20** [comment]
Make sure to fix all the lint errors here:
image

File: new-dti-website/src/app/apply/apply-timeline.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
+
+/* eslint-disable no-param-reassign */
```
Comment (henry-li-06) [inline new-dti-website/src/app/apply/apply-timeline.tsx:2]: :(

Comment (henry-li-06) [reply new-dti-website/src/app/apply/apply-timeline.tsx:2]: https://stackoverflow.com/questions/35637770/how-to-avoid-no-param-reassign-when-setting-a-property-on-a-dom-object

**Comment by oscarwang20** [comment]
Make sure to follow the [designs](https://www.figma.com/file/jKbvR1jVr1k80GGWnCpiRT/Website-Designs-SP23%2FFA23?type=design&node-id=6005%3A28150&mode=design&t=2WKgP0Vlx4VXvPO0-1) for the tablet and mobile views as well to ensure that the component is responsive to changes in screen size. Use the tailwind breakpoints for `lg`, `md`, and `sm` to use different styles based on the current screen width. Check out `navbar.tsx` for an example and lmk if you need any help!

**Review by cchrischen — COMMENTED** [review]
Great work so far, Kevin! :) The responsiveness still seems a bit off from the Figma designs, when I go to tablet mode. For example, the "Down to Innovate" blurb gets pushed down a screen height. Also, one thing I learned is that Tailwind supports custom inputs, which you might want to consider doing. I left some comments talking about this. 

Another note: To keep the same organization as IDOL frontend and the other pages of this website, I think it would be best for `ApplyTimeline` to be put in an apply subfolder of components.

File: new-dti-website/src/app/apply/apply-timeline.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      location: 'Kimball Hall B11'
+    },
+    {
+      title: 'APPLICATIONS DUE!',
+      content:
+        'We’re welcoming any and all students who are looking to make a difference through tech. Applicants are not considered on a rolling basis.',
+      date: '2024-04-01',
+      location: 'Online Submission'
+    }
+  ];
```
Comment (cchrischen) [inline new-dti-website/src/app/apply/apply-timeline.tsx:37]: For the other pages, we put these large objects in a separate json file in the same directory. Consider doing this too -- it'll make editing application timelines easier in the future, too!

File: new-dti-website/src/app/apply/page.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          style={{
+            backgroundColor: '#A52424',
+            color: 'white',
+            padding: '8px 16px',
+            fontSize: '16px md:18px',
+            borderRadius: '10px',
+            border: 'none',
+            cursor: 'pointer',
+            marginTop: '1em',
+          }}
```
Comment (cchrischen) [inline new-dti-website/src/app/apply/page.tsx:50]: Many of the in-line styling can be converted into Tailwind. This button element can have `className=bg-[#A52424]...`, which will accomplish the same thing.

File: new-dti-website/src/app/apply/apply-timeline.tsx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      style={{
+        backgroundImage: "url('/images/apply_timeline_bg.png')",
+        backgroundSize: 'cover'
+      }}
+    >
+      <div className="text-white w-full md:px-0" style={{ alignSelf: 'flex-start' }}>
+        <h1 className="text-4xl font-semibold mb-4">
+          This is DTI.
+        </h1>
+        <p className="text-3xl font-semibold mb-4">developing, designing, delivering.</p>
```
Comment (cchrischen) [inline new-dti-website/src/app/apply/apply-timeline.tsx:77]: nit: the d in developing should be capitalized

**Comment by andrew032011** [comment]
@kevinmram could you also rebase against main or merge main into this branch? The branch has fallen a bit behind and the deploy preview won't work until we pick up the node downgrade.

**Comment by andrew032011** [comment]
@cchrischen is taking on this change and will open up a new PR!

**Pull request closed without merging**